### DISCUSSION TITLE
Change colorscheme with different file

### DISCUSSION THREAD

**Discussion by bengidev** [LazyVim/LazyVim#2350] [Q&A]
Hello, i want to change colorscheme into Catppuccin plugin with creating a new file colorscheme.lua, and after i save the changes, it seems doesn't work. Am i missing something?

![Screenshot 2024-01-13 at 12 45 04](https://github.com/LazyVim/LazyVim/assets/48014755/781ea4b7-8a7b-4c52-855c-396f4c16a269)
![Screenshot 2024-01-13 at 12 47 08](https://github.com/LazyVim/LazyVim/assets/48014755/17722fac-4337-4126-a42c-8f688f9cbbc8)

Appreciate your help, thanks 🍻

**Comment by dpetka2001** [comment]
The way to change colorscheme is described in the [docs](https://www.lazyvim.org/configuration#icons--colorscheme). Please do take some time to read them as they pertain quite useful information. In short you could create a file `~/.config/nvim/lua/plugins/colorscheme.lua` with following contents
```lua
return {
  {
    "LazyVim/LazyVim",
    opts = {
      colorscheme = "catppuccin",
    },
  },
}
```

**Reply by VesperQuartz** [reply]
> colorscheme.lua

why must the file specifically be named colorscheme.lua cause i use the name core.lua and it didn't work
